content/en/_index.md

```diff
@@ -110,10 +117,20 @@ public static void main(String[] args) {
 }
 
 // This is an example snippet for showing best practices.
-public static void exampleSnippet(String projectId, String filePath) {
+public static void inspectImageFile(String projectId, String filePath) {
     // Snippet content ...
 }
 {{< /tab >}}
+{{< tab header="Python" >}}
+def inspect_image_file(project_id: str, file_path: str):
+    # Snippet content ...
+
+def main():
```

Comment:
IMO, I think we should lead with main() here so it's the first thing the user sees.

At the bottom I would suggest putting in:
```python
if __name__ == '__main__':
     main()
```

Reply:
Switched order and added `if __name__ == ...` block.